src/Umbraco.Web/Editors/GravatarController.cs

```diff
@@ -13,6 +14,14 @@ public class GravatarController : UmbracoAuthorizedJsonController
     {
         public string GetCurrentUserGravatarUrl()
         {
+            // If FIPS is required, never check the Gravatar service as it only supports MD5 hashing.  
+            // Unfortunately, if the FIPS setting is enabled on Windows, using MD5 will throw an exception
+            // and the website will not run.
+            if (CryptoConfig.AllowOnlyFipsAlgorithms)
+            {
+                return null;
```

Comment:
No JS errors present and the request returns a status of 200.  The image simply stays a teal "U" Umbraco logo.